lib/workers/pr/changelog/gitlab/index.ts

```diff
@@ -107,14 +108,7 @@ export async function getReleaseList(
   )}projects/${repoId}/releases`;
 
   // TODO: use type from PR #11226
```

Comment:
Is this TODO still relevant now that you've removed/edited the previous code?

Reply:
```suggestion
```